### PR TITLE
MMB: Add fill travel document detail screen

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: packages/navigation/types/Navigation.js [hunk 1]
```diff
@@ -40,7 +40,8 @@ type MMBStackNavigatorNames =
   | 'mmb.tickets.boarding_pass'
   | 'mmb.timeline.invoice'
   | 'mmb.explore'
-  | 'DetailScreen';
+  | 'DetailScreen'
+  | 'TravelDocumentScreen';
```
Comment (RobinCsl) [inline packages/navigation/types/Navigation.js]: Just out of curiosity: why do we have a different naming convention here?

Comment (tbergq) [reply packages/navigation/types/Navigation.js]: Not really sure 🤷‍♂️

Comment (RobinCsl) [reply packages/navigation/types/Navigation.js]: I believe it would be better to stick to one, to avoid confusions in the future. What do you think?

Comment (tbergq) [reply packages/navigation/types/Navigation.js]: Well, this dotted syntax is different somehow. They are actual screens on mobile, but not on ios

Comment (RobinCsl) [reply packages/navigation/types/Navigation.js]: Ok then. If you think that it makes sense to have this distinction, I'll approve it.

**Pull request merged**